src/serializers/encoding_version.h

```diff
@@ -6,7 +6,7 @@
 
 #pragma once
 
-#define GRAPH_ENCODING_VERSION_LATEST 10  // Latest RDB encoding version.
+#define GRAPH_ENCODING_VERSION_LATEST 11  // Latest RDB encoding version.
```

Comment:
When was decoder version 7 was introduced? what was the RedisGraph version at that time?

Reply:
2.2.8